Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/proxy/AuditRepositoryProxyWebServiceUnsecuredImpl.java

```diff
@@ -45,42 +44,48 @@
  * @author akong
  */
 public class AuditRepositoryProxyWebServiceUnsecuredImpl implements AuditRepositoryProxy {
+
     private static final Logger LOG = Logger.getLogger(AuditRepositoryProxyWebServiceUnsecuredImpl.class);
 
-    private WebServiceProxyHelper oProxyHelper = new WebServiceProxyHelper();
+    private final WebServiceProxyHelper oProxyHelper = new WebServiceProxyHelper();
+    private AcknowledgementType result = new AcknowledgementType();
 
+    @Override
     public AcknowledgementType auditLog(LogEventRequestType request, AssertionType assertion) {
+
         LOG.debug("Entering AuditRepositoryProxyWebServiceUnsecuredImpl.auditLog(...)");
```

Comment:
Can this (and other instances of "Entering Class.Method") either be changed to trace level, or removed in favor of a more meaningful log message?